reference/format/dashboard.rst

```diff
@@ -0,0 +1,101 @@
+Dashboard
+=========
+
++-----------+--------------------------+
+| Filename  | ``dashboard.yml``        |
++-----------+--------------------------+
+| Root Node | ``oro_dashboard_config`` |
++-----------+--------------------------+
+| Sections  | * `dashboards`_          |
+|           |                          |
+|           |   * `twig`_              |
+|           |                          |
+|           | * `widgets`_             |
+|           |                          |
```

Comment:
This table if hard to understand because this structure does not show the real dashboard.yml file tree structure. Please add an example of this file to better understading.